components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <article class="bx--grid page-section qiskit-capabilities-section">
+    <h2 v-text="title" />
+  </article>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class TheQiskitCapabilitiesSection extends Vue {
+  @Prop(String) title!: string
+  @Prop(String) description!: string
+}
+</script>
+
+<style lang="scss" scoped>
+.qiskit-capabilities-section {
+  // todo: add styles
+}
+</style>
```

Comment:
I believe we shouldn't include this just now as it has no effect and we're also not 100% we are going to need scoped styles in this component, yet.

So it'd be better to just ship what the components needs to work at the moment and include the styles later as needed in the following PR.

```suggestion
```